### DISCUSSION TITLE
convet OUTCAR to multiframe-lmp dump data

### DISCUSSION THREAD

**Discussion by Wangmumu0403** [deepmodeling/dpdata#329] [Q&A]
this code is only get one frame ,if I want to get multiframe from OUTCAR , how can I get it ?

<img width="915" alt="image" src="https://user-images.githubusercontent.com/95537862/174620929-8fc1e7f1-eb1f-4b05-8a48-1b9a1267f812.png">

**Comment by njzjz** [comment]
You may remove `frame_idx=0`

**Reply by Wangmumu0403** [reply]
I have removed it yet  it did not help . the result is still one frame.

**Reply by namtranlab** [reply]
did you try a loop?
